src/app/components/BookList/BookList.jsx

```diff
@@ -10,18 +10,19 @@ const BookList = ({
   picks,
   type,
   displayInfo,
+  displayType,
   picksCount,
 }) => {
   const renderBookItems = currentBooks => (
     currentBooks.length ?
-      currentBooks.map(book => <Book key={book.slug} pick={book} isJsEnabled={isJsEnabled} />)
```

Comment:
If we delete `key` here it will still work, but we will receive the complaints from React on bowser console.